recipe/gcc_stdmacro.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/src/wasm/CMakeLists.txt b/src/wasm/CMakeLists.txt
+index d24618c63..28cb3aab1 100644
+--- a/src/wasm/CMakeLists.txt
++++ b/src/wasm/CMakeLists.txt
+@@ -14,4 +14,6 @@ set(wasm_SOURCES
+   wasm-validator.cpp
+   ${wasm_HEADERS}
+ )
++# needed for gcc to compile the llvm sources included here
++add_definitions(-D__STDC_FORMAT_MACROS)
```

Comment:
Instead of patching, you can set it in `CXXFLAGS` and `CFLAGS` env variables

Reply:
Oops, sorry didn't see this before. Making this change also makes sense.